content/amp_amg/software.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Install and Configure Tools"
+chapter: false
+weight: 5
+---
+
+In the Cloud9 workspace, run the following commands:
+
+## Install and setup prerequisites
+
+```
+# Install prerequisite packages
+sudo yum -y install jq nodejs python37
+
+# Setting CDK Version
+export AWS_CDK_VERSION="1.41.0"
+
+# Install aws-cdk
+npm install -g --force aws-cdk@$AWS_CDK_VERSION
+
+# Install cdk packages
+pip3 install --user --upgrade aws-cdk.core==$AWS_CDK_VERSION \
+aws-cdk.aws_ecs_patterns==$AWS_CDK_VERSION \
```

Comment:
Are all of these modules needed? Can we scope this down?

Reply:
This is basically platform part to build eco cluster, same as in other modules.